### ISSUE TITLE
Discovery and device topics are hardcoded

### ISSUE THREAD

**Issue by mikeS7** [shaonianzhentan/node-red-contrib-ha-mqtt#24]
Could you please make discovery and device topics configurable.

Default values 'homeassistant' and 'ha-mqtt' are hardcoded.

**Comment by DirkMu** [comment]
Same for 'identifiers' and 'sw_version'.

**Comment by mikeS7** [comment]
> Same for 'identifiers' and 'sw_version'.

you can overwrite in device configuration:
`{
    "configuration_url": "http://xxx.yyy:1880/",
    "identifiers": "specialId",
    "manufacturer": "SuperPuper",
    "model": "Greatest-Sensor",
    "sw_version": "0.0.1"
}`

**Comment by DirkMu** [comment]
Yes, you are right. It was not updated initially, yesterday. I re-tested it and finally I got it working. Thanks for the hint.

**Comment by rlust** [comment]
Has anyone been able to create a light device?
